app/plugins/hmac-authentication.md

```diff
@@ -72,6 +72,10 @@ params:
 In order to use the plugin, you first need to create a Consumer to associate
 one or more credentials to.
 
+Note: Please make sure that all the request parameters being used by client to
```

Comment:
Looks like this sentence has some some grammatical issues? We should maybe explain _why_ as well.
```
Because the HMAC signature is generated by the client, you should make sure that Kong does not
update or remove any signed request parameter before this plugin runs.
```

Or something like that...